tedge/src/cli/connect/command.rs

```diff
@@ -124,13 +129,15 @@ impl ConnectCommand {
     }
 
     fn check_connection(&self) -> Result<(), ConnectError> {
```

Comment:
try passing in `config` as parameter instead of using the config repository....

Reply:
done